CMSgov/design-system
# [WNMGDS-2250] Add primary util classes for bg color

## Summary

- There were a few bg color utility classes missing for primary color that have now been added

## How to test

1. Run `yarn start` and visit the doc site page for the utility bg colors and see the full set of primary colors represented 

## Checklist




### If this is a change to code:


### If this is a change to documentation:
